kms/api-client/snippets_test.py

```diff
@@ -177,31 +179,39 @@ def test_key_policy(self):
                                                  self.symId,
                                                  self.member,
                                                  self.role)
-        policy = snippets.get_crypto_key_policy(self.project_id,
-                                                self.location,
-                                                self.keyring_id,
-                                                self.symId)
-        found = False
-        for b in list(policy.bindings):
-            if b.role == self.role and self.member in b.members:
-                found = True
-        assert found
+
+        def check_policy():
+            policy = snippets.get_crypto_key_policy(self.project_id,
+                                                    self.location,
+                                                    self.keyring_id,
+                                                    self.symId)
+            found = False
+            for b in list(policy.bindings):
+                if b.role == self.role and self.member in b.members:
+                    found = True
+            assert found
+        eventually_consistent.call(check_policy,
```

Comment:
Unfortunately the decorator form doesn't work with the current 0.0.15 release.
See: https://github.com/GoogleCloudPlatform/python-repo-tools/issues/25

I'm trying to make a new release of the above module. 

I'm fine with the current form :)